extensions/ql-vscode/src/view/variant-analysis/VariantAnalysisSkippedRepositoryRow.tsx

```diff
@@ -0,0 +1,48 @@
+import { VSCodeBadge, VSCodeCheckbox } from '@vscode/webview-ui-toolkit/react';
+import * as React from 'react';
+import styled from 'styled-components';
+import { WarningIcon } from '../common';
+import { VariantAnalysisSkippedRepository as SkippedRepo } from '../../remote-queries/shared/variant-analysis';
+
+export type VariantAnalysisSkippedRepositoryRowProps = {
+  repository: SkippedRepo,
+};
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 0.5em;
+  align-items: center;
+`;
+
+const ChevronIcon = styled.span`
+  color: var(--vscode-descriptionForeground);
+`;
```

Comment:
Instead of using a `span`, you should be able to use the `Codicon` component:

```suggestion
const ChevronIcon = styled(Codicon)`
  color: var(--vscode-descriptionForeground);
`;
```